src/TeamPage/__test__/teamPage.stories.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import TeamPage from '../teamPage';
+import { Navbar } from '../../Navbar/navbar';
+import { Footer } from '../../Footer/footer';
+
+const footerActions = {
+  onLogIn: action('logging in'),
+  onLogOut: action('logging out'),
+};
+
+storiesOf('Team Page', module)
+  .addDecorator(story => (
+    <div className="App">
+      <header>
+        <Navbar loggedIn />
```

Comment:
`loggedIn` had been changed to `isAuthenticated`.

Reply:
changes made in navbar and footer.